crates/ruma-state-res/src/power_levels.rs

```diff
@@ -0,0 +1,105 @@
+use std::collections::BTreeMap;
```

Comment:
Nice, thanks for breaking the PL deserialization logic into its own module.

Reply:
I should probably also move `IntPowerLevelsEventContent` there to be thorough...